components/contribution-flow/StepDetailsCrypto.js

```diff
@@ -42,9 +84,10 @@ const StepDetailsCrypto = ({ onChange, data }) => {
       <StyledSelect
         inputId="crypto-currency"
         options={CRYPTO_CURRENCIES}
-        defaultValue={currencyType}
-        onChange={value => {
-          setCurrencyType(value);
+        defaultValue={cryptoCurrencyType}
+        onChange={async value => {
```

Comment:
`value` is a bit confusing, as your setting it to this force you doing `cryptoCurrency.value` later, aka `value.value`. Which is difficult to process.

Maybe you wanted `onChange={(selectedCryptoCurrency)` or `onChange={(cryptoCurrency)` ?

Reply:
`async` not needed